tests/jquery.flot.navigate.Test.js

```diff
@@ -509,7 +397,210 @@ describe("flot navigate plugin", function () {
+        simulate.touchmove(canvasElement, pointCoords[1].x, pointCoords[1].y);
+        simulate.touchend(canvasElement, pointCoords[1].x, pointCoords[1].y);
+
+        expect(xaxis.min).toBeCloseTo(initialXmin + (canvasCoords[0].x - canvasCoords[1].x), 6);
+        expect(xaxis.max).toBeCloseTo(initialXmax + (canvasCoords[0].x - canvasCoords[1].x), 6);
+        expect(yaxis.min).toBeCloseTo(initialYmin + (canvasCoords[0].y - canvasCoords[1].y), 6);
+        expect(yaxis.max).toBeCloseTo(initialYmax + (canvasCoords[0].y - canvasCoords[1].y), 6);
+      });
+
+      it('should drag the logarithmic plot', function() {
+          var d1 = [];
+          for (var i = 0; i < 14; i += 0.2) {
+              d1.push([i, 1.01 + Math.sin(i)]);
+          }
+
+          var plot = $.plot(placeholder, [d1], {
+              series: {
+                  lines: { show: true },
+                  points: { show: true }
+              },
+              xaxis: { autoscale: 'exact' },
+              yaxis: { mode: 'log', showTickLabels: "all", autoscale: 'exact' },
+              zoom: { interactive: true },
+              pan: { interactive: true, enableTouch: true }
+          });
+
+          var canvasElement = placeholder[0].childNodes[2],
+              xaxis = plot.getXAxes()[0],
+              yaxis = plot.getYAxes()[0],
+              initialXmin = xaxis.min,
+              initialXmax = xaxis.max,
+              initialYmin = yaxis.min,
+              initialYmax = yaxis.max,
+              canvasCoords = [ { x : 4, y : 0.7 }, { x : 2, y : 10 } ],
+              pointCoords = [
+                      getPairOfCoords(xaxis, yaxis, canvasCoords[0].x, canvasCoords[0].y),
+                      getPairOfCoords(xaxis, yaxis, canvasCoords[1].x, canvasCoords[1].y)
+              ];
+
+          simulate.touchstart(canvasElement, pointCoords[0].x, pointCoords[0].y);
+          simulate.touchmove(canvasElement, pointCoords[1].x, pointCoords[1].y);
+          simulate.touchend(canvasElement, pointCoords[1].x, pointCoords[1].y);
+
+          expect(xaxis.min).toBeCloseTo(initialXmin + (canvasCoords[0].x - canvasCoords[1].x), 6);
+          expect(xaxis.max).toBeCloseTo(initialXmax + (canvasCoords[0].x - canvasCoords[1].x), 6);
+          expect(yaxis.min).toBeCloseTo((yaxis.c2p(yaxis.p2c(initialYmin) + (pointCoords[0].y - pointCoords[1].y))), 6);
+          expect(yaxis.max).toBeCloseTo((yaxis.c2p(yaxis.p2c(initialYmax) + (pointCoords[0].y - pointCoords[1].y))), 6);
+
+      });
+
+      it('should drag the point in the same way for many sequential moves as for one long move',function() {
+
+         //deactivate ticks for precision
+        options.yaxes[0].showTickLabels = 'none';
+        options.xaxes[0].showTickLabels = 'all';
+
+        plot = $.plot(placeholder, [
+            [
+                [-10, -10],
+                [120, 120]
+            ]
+        ], options);
+
+        var canvasElement = placeholder[0].childNodes[2],
```

Comment:
@cipix2000 testul pentru panning doar cu axis.min/max pentru updatarea offsetului in pan.